lib/dmv_facility.rb

```diff
@@ -0,0 +1,28 @@
+class DmvFacility
+  def create_facilities(databases)
+    datum = databases.map do |data|
+      Facility.new({
+        name: access_name(data),
+        address: access_address(data),
+        phone: access_phone(data),
+      })
+    end
+  end
+
+  # require 'pry'; binding.pry
+  def access_name(datum)
+    datum[:title]
+  end
+
+  def access_address(datum)
+    datum.include?("oregon")
+      address_data = eval(datums[:location_1][:human_address])
+      address = address_data.fetch_values(:address, :city, :state, :zip).join(" ")
+    end
+
+  def access_phone(data)
+    data.include?("oregon")
```

Comment:
It isn't very conventional to have this conditional `data.include?("oregon")` just hanging out by itself. Do you want it in an if statement or a guard clause?